simespresso/io_QE/tests/test_write_input.py

```diff
@@ -0,0 +1,71 @@
+__author__ = 'jeremy'
+
+import unittest
+import subprocess
+import logging
+import sys
+
+from simphony.core.data_container import DataContainer
+from simphony.core.cuba import CUBA
+from simphony.cuds.particles import Particle, Particles
+from simespresso.io_QE import espresso_class
+#import simespresso.io_QE.espresso_class
+#import io_QE.espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
ok - is camelCase  the simphony standard for filenames? iiuc , lowercase_with_underscores is standard for python